scipy/special/tests/test_basic.py

```diff
@@ -792,13 +792,6 @@ def test_nctdtrinc(self):
     def test_nctdtrit(self):
         cephes.nctdtrit(.1,0.2,.5)
 
-    def test_ndtr(self):
```

Comment:
Why just these tests being moved?  Is it the long-term goal to move all `cephes` tests out of `test_basic::TestCephes` and into their own files?

Reply:
Generally speaking we've gradually been moving non-mpmath tests for groups of functions into their own files. I'd say the overall goal is to expand on the tests in `test_basic`, and once expanded they tend to fit better in their own file so that we don't get a thousands-of-lines-long `test_basic.py`. A reasonable example of an "expanded" set of basic tests is:

https://github.com/scipy/scipy/blob/master/scipy/special/tests/test_loggamma.py